src/components/chat/ChattingScreen.tsx

```
@@ -0,0 +1,90 @@
+          </div>
+        ))}
+      </div>
+      <div className="px-5">
+        <Button
+          variant="solid"
+          tone="secondary"
+          size="l"
+          children={
+            <div className="flex">
+              <PhoneIcon className="mr-1" />
+              상담사 전화 연결
```

Biome (2.1.2)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)